ethpm_cli/commands/install.py

```diff
@@ -119,6 +129,93 @@ def get_package_aliases(package_name: str, config: Config) -> Iterable[Tuple[str
                 yield alias
 
 
+def resolve_pkg_name_or_alias(pkg_name: str, config: Config) -> InstalledPackage:
+    lockfile_path = config.ethpm_dir / "ethpm.lock"
+    lockfile = json.loads(lockfile_path.read_text())
+    return InstalledPackage(**lockfile[pkg_name])
+
+
+def update_package(args: Namespace, config: Config) -> None:
+    if is_package_installed(args.package, config):
```

Comment:
You might invert this so that you do an early exit in the event of what is currently the `else` clause, and then you can dedent this entire large block of code under the `if`.  I'd call this a minor *code smell*, having such a huge block of code inside of an `if` statement like this.